### PR TITLE
French translations

### PR DESCRIPTION
**Pull request by Bouboufez**
Here are the files for the french translations of the bot

### CODE REVIEW

File: i18n/8ball/fr.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  "a10": "Les signes indiquent oui .",
+  "u1": "Réponse vague, réessayez.",
+  "u2": "Redemandez plus tard.",
+  "u3": "Il vaut mieux ne pas vous le dire maintenant.",
+  "u4": "Je ne peux pas le prédire maintenant.",
+  "u5": "Concentrez-vous et redemandez.",
+  "n1": "Ne comptez pas là-dessus.",
+  "n2": "Ma réponse est non.",
+  "n3": "Mes sources disent non.",
+  "n4": La perspective n'est pas si bonne.",
```
Comment (Kenny2github) [inline i18n/8ball/fr.json]: Missing a quote at the start of this string.
add `"` before `La persp...`

File: i18n/battleship/fr.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
+  "place-instructions": "Cliquez sur un bouton **deux fois** pour déplacer le bateau ; cliquez sur le bouton vert deux fois lorsque c'est bon.",
+  "edit-instructions": "Merci d'envoyer un message contenant le contenu exact :",
+  "fire-instructions": "Modifiez le message \"EDITEE\" message pour obtenir les coordonnées (ex : A0 ou 3b) où tirer.",
+  "wait-instructions": "En attente que l'autre joueur tire",
+  "won-title": Tu as gagné !",
```
Comment (Kenny2github) [inline i18n/battleship/fr.json]: Missing a quote at the start of this string.
add `"` before `Tu as...`

File: i18n/card_games/fr.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  "fish-checkbooks-title": "Nouveaux livress",
+  "fish-m-card": "Envoyez le numéro d'une carte (ex : Q ou 10) pour la demander.",
+  "fish-m-wait": "En attente de la question du joueur suivant...",
+  "fish-turn": "C'est actuellement le tour de {0}.",
+  "fish-card-taken": "{0} a demandé un {1} - puisque vous avez un {2}, vous êtes obligés de le lui donner.",
+  "fish-card-got": "Vous avez : {0}",
+  "fish-card-missed-unsafe": "{0} a demandé un {1} - bien que vous ayez un {2}, vous n'êtes pas encore en sécurité.",
+  "fish-card-missed": "{0} a demandé un {1} - puisque vous avez un {2}, vous êtes en sécurité.",
+  "fish-fish-got": "Allez pêcher ! Vous avez un {0} - c'est ce que vous aviez demandé !",
+  "fish-fish": Allez pêcher ! Vous avez un {0}.",
```
Comment (Kenny2github) [inline i18n/card_games/fr.json]: Missing a quote at the start of this string.
add `"` before `Allez...`

File: i18n/connect4/fr.json [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+  "config-nothing-set": "Ce salon n'a aucune configuration ! Exécutez ;help connect4_config pour savoir comment le configurer.",
+  "config-all-set": "Salon pour les parties globales de puissance 4 : {0}\nRôle à mentionner : {1}",
+  "config-channel-set": "Salon pour les parties globales de puissance 4 : {0}",
+  "config-reset": "La configuration de ce serveur a été retirée avec succès.",
+  "config-no-empty": "Impossible de configurer sans saisir de paramètres...",
+  "config-set-channel": "Le salon pour les parties globales de puissance 4 a été configuré avec succès sur {0}",
+  "config-no-channel-but-role": "Impossible de paramétrer une mention pour les parties globales de puissance 4 sans paramétrer un salon !",
+  "config-set-role": "Le salon pour les parties globales de puissance 4 a été configuré avec succès sur {0} et le rôle à mentionner a été configuré sur {1}",
+  "ready-player-self": "Joueur 2 : {0}",
+  "ready-player-global": En attente que quelqu'un d'un autre serveur rejoigne...",
```
Comment (Kenny2github) [inline i18n/connect4/fr.json]: Missing a quote at the start of this string.
add `"` before `En attente...`

File: i18n/connect4/fr.json [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+  "board-turn": "C'est actuellement le tour de {0}.",
+  "board-title": "Tableau",
+  "board-you-are": "\nVous êtes {0}",
+  "game-timeout": Personne n'a fait dde mouvement depuis {0} secondes ! le jeu a été annulé.",
+  "game-over-title": "Fin de la partie !",
+  "game-over-tied": "Egalité ! Le tableau est plein et aucune rangée de 4 n'a été faite.",
+  "game-over-description": "Fin de la partie !\Gagnant: {0}{1}\nPerdant: {2}{3}"
```
Comment (Kenny2github) [inline i18n/connect4/fr.json]: You cut out the `n` in `\n`.
Change `partie !\Gagnant:` to `partie !\nGagnant:`

File: i18n/pm_games/fr.json [hunk 6]
```diff
@@ -0,0 +1,13 @@
+{
+  "playing": "En train de jouer à {0}",
+  "ready-player-1": "Joueur 1 : {0}\n",
+  "react-to-join": "Réagissez avec {0} pour rejoindre !",
+  "ready-player-2": "Joueur 2 : {0}\nEn attende qu'un second joueur rejoigne...",
+  "game-timeout": "Personne n'a joué depuis {0} secondes ! Le jeu a été automatiquement annulé.",
+  "game-cancelled": "Jeu annulé par celui qui l'a lancé.",
+  "no-bots": "Attendez, vous ne pouvez pas jouer contre un bot ! Jeu annulé.",
+  "no-offline": "Attendez, joueur 2 est déconnecté ! Jeu annulé.",
+  "unready-player-2": Le joueur 2 n'a pas rejoint après {0} secondes ! Le jeu a été automatiquement annulé.",
```
Comment (Kenny2github) [inline i18n/pm_games/fr.json]: Missing a quote at the start of this string.
add `"` before `Le joueur...`

File: i18n/scratch/fr.json [hunk 7]
```diff
@@ -0,0 +1,4 @@
+{
+  "messagecount-failed": Impossible de vérifier le nombre de messages de {0}",
```
Comment (Kenny2github) [inline i18n/scratch/fr.json]: Missing a quote at the start of this string.
add `"` before `Impossible...`

File: i18n/wiki/fr.json [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+  "iwstats-time": "Temps",
+  "iwstats-pages": "Pages",
+  "iwstats-articles": "Articles",
+  "iwstats-edits": "Modifications",
+  "iwstats-images": "Images",
+  "iwstats-users": "Utilisateurs",
+  "iwstats-activeusers": "Utilisateurs actifs",
+  "iwstats-admins": "Admins",
+  "iwstats-jobs": "Travail",
+  "wikistats-failed": Echec lors de la récupération des statistiques. Vous avez certainement spcéifié un wiki qui n'existe pas.",
```
Comment (Kenny2github) [inline i18n/wiki/fr.json]: Missing a quote at the start of this string.
add `"` before `Echec...`

**Review by Kenny2github — CHANGES_REQUESTED** [review]
A lot of syntax errors.

**Comment by Bouboufez** [comment]
I've fixed all the problems
Sorry

**Review by Kenny2github — APPROVED** [review]

**Pull request merged**